chainer/link.py

```diff
@@ -358,24 +358,59 @@ def to_gpu(self, device=None):
         Args:
             device: Target device specifier. If omitted, the current device is
                 used.
+            sticky (bool): If ``True``, the link will be transferred to the
```

Comment:
Looks the description for `True` is actually for `False` (and vice versa for the other one)?

Reply:
Sorry, you're right. Fixed.